ga4gh/serverconfig.py

```diff
@@ -39,7 +39,7 @@ class DevelopmentConfig(BaseConfig):
     Configuration used for development.
     """
     DATA_SOURCE = "ga4gh-example-data"
-    DEBUG = False
+    DEBUG = True
```

Comment:
I think this causes problems for people. Best leave it False by default (people can turn it on temporarily if they want).